recipes/scs/meta.yaml

```diff
@@ -0,0 +1,46 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "1.2.6" %}
+# {% set sha256 = "4a40c81b9d1ab4f1617c106dc103dea0ae5aa553ba29bbc3ada5b259c45e4a1f" %}
+
+package:
+  name: scs
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/cvxgrp/scs.git  # we need to use git master because of experimental compilation feature
+
```

Comment:
If you need more arguments than those for the install step, please add what you need.